README.md

```diff
@@ -77,85 +63,20 @@ git clone https://github.com/PegaSysEng/teku.git
 
 ### Other Useful Gradle Targets
 
-| Target       |  Description                              |
+| Target       | Builds                              |
 |--------------|--------------------------------------------
-| distTar      | Builds a full distribution in build/distributions (as .tar.gz)
-| distZip      | Builds a full distribution in build/distributions (as .zip)
-| installDist  | Builds an expanded distribution in build/install/teku
-| distDocker   | Builds the pegasyseng/teku docker image
-
-## Run Multiple Teku nodes
-
-### Prereqs:
-
-- tmux
-
-After building with `./gradlew distTar`, the simplest way to run is by using this command: 
-
-```shell script
-cd scripts && sh run.sh -n=[NUMBER OF NODES]
-```
-
-Help is available for this script as well:
-
-```
-sh run.sh -h
-Runs a simulation of Teku with NODES nodes, where NODES > 0 and NODES < 256
-Usage: sh run.sh [--numNodes, -n=NODES]  [--config=/path/to/your-config.yaml] [--logging, -l=OFF|FATAL|WARN|INFO|DEBUG|TRACE|ALL]
-                 [--help, -h]
-- If config files are specifed for specific nodes, those input files will be used to
-configure their respective nodes.
-- If no logging option is specified, then INFO is the default
-```
+| distTar      | Full distribution in build/distributions (as `.tar.gz`)
+| distZip      | Full distribution in build/distributions (as `.zip`)
+| installDist  | Expanded distribution in `build/install/teku`
+| distDocker   | The `pegasyseng/teku` docker image
 
-## Run in Interop Mode
+## Configuration Defaults 
 
-An interop script is provided to create a network with Teku and a number of other clients. 
-
-### Prereqs:
-
-- tmux
-- [zcli](https://github.com/protolambda/zcli)
-- You will need to have at least one other client built on your machine for this to work
-
-The simplest way to run in interop mode is by using this command: 
-
-```shell script
-cd scripts
-sh interop.sh [validator_count] [owned_validator_start_index] [owned_validator_count] [start_delay]
-```
-Help is available for this script as well:
-
-```
-sh interop.sh 
-Runs a multiclient testnet
-Usage: sh interop.sh [validator_count] [owned_validator_start_index] [owned_validator_count] [start_delay]
-Example: Run multiple clients in interop mode using static peering. 16 validators and all are assigned to Teku
-         sh interop.sh 16 0 16 10
-```
-
-### Manual configuration
-
-To configure it manually, set these options in the config.yaml:
-```yaml
-# interop
-# when genesis time is set to 0, teku takes genesis time as currentTime + 5 seconds.
-#seconds since 1970-01-01 00:00:00 UTC
-Xinterop-genesis-time: 5778872 
-Xinterop-owned-validator-start-index: 0
-Xinterop-owned-validator-count: 8
-Xinterop-start-state: "/tmp/genesis.ssz"
-Xinterop-number-of-validators: 16
-Xinterop-enabled: True
-```
-
-## Deposit Simulation
-
-Click [here](pow/README.md) for setup instructions
+Refer to `config/config.yaml` for a set of default configuration settings.
```

Comment:
This is no longer correct.  `--help` is now the best way to see the default config settings.  `config/config.yaml` is only used by `run.sh` I think.

Reply:
Fixed